app/main.py

```diff
@@ -1,6 +1,41 @@
+import os
+
+import requests
+
+URL = "https://api.weatherapi.com/v1/current.json"
+
+API_KEY = os.getenv("API_KEY")
+
+PARAMS = {
+    "key": API_KEY,
+    "q": "Paris",
+}
+
+
+def get_res() -> dict:
+    try:
+        print("Performing request to Weather API to city Paris...")
+        response = requests.get(URL, params=PARAMS)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        print(f"HTTP ERROR: {err}")
+    except requests.exceptions.RequestException as err:
+        print(f"REQUEST ERROR: {err}")
+    return response.json()
```

Comment:
The `return response.json()` statement is outside the `try` block, which means it will execute even if an exception is raised. This could lead to a `NameError` if `response` is not defined due to an exception. Consider returning a default value or handling this case appropriately.